packages/kbn-handlebars/README.md

```diff
@@ -0,0 +1,173 @@
+# @kbn/handlebars
+
+A custom version of the handlebars package which, to improve security, does not use `eval` or `new Function`. This means that templates can't be compiled in advance and hence, rendering the templates is a lot slower.
```

Comment:
In your PR description, you mentioned this:

> ## Limitations
> * Only the following compile options will be supported:
>   
>   * `knownHelpers`
> ...

It would probably be a good idea to include that info in this README, too :+1:

Reply:
Fixed in https://github.com/elastic/kibana/pull/124484/commits/7ff84e51cc459eb4ff6dff5b7cb4f4781429ca23